modules/PowermeterBSM_WS36A/doc.rst

```diff
@@ -0,0 +1,22 @@
+.. _everest_modules_handwritten_PowermeterBSM_WS36A:
+
+..  This file is a placeholder for an optional single file handwritten documentation for 
```

Comment:
It seems like you did neither :). I think you should put a few lines of documentation here: Why this module, how it differers vs the existing and how it interacts with the Serial one.